src/GetSet.php

```diff
@@ -9,11 +9,12 @@
 namespace Revinate\GetterSetter;
 
 
-class GetSet implements GetSetInterface {
+class GetSet implements GetSetInterface
+{
```

Comment:
So this is the new CS? class "{" is on the next line, while function "{" is on the same line?

Reply:
oh crap. sorry. left this out of sight